### ISSUE TITLE
Logical view: Remove blinking red animation for invalid outputs

### ISSUE THREAD

**Issue by Luke18033** [ChaosInitiative/Portal-2-Community-Edition#456]
## Which component should be improved?
Hammer

## Describe your suggestion
In the newly-unhidden logical view, outputs marked as invalid will flash red, which is a little annoying considering that they may not actually be invalid (for things like instances or special targetnames). It'd be nice to have this removed.

## Expected result
The outputs should appear as a solid red color, with no blinking animation.

**Comment by Luke18033** [comment]
Actually, instance I/O getting marked as invalid would probably be a bug itself.

**Comment by CitadelCore** [comment]
Yeah. This is not a bug. Please open an issue for the instance I/O issue.

**Comment by Luke18033** [comment]
That issue's already open I think. And this was a suggestion, not a bug. There are still other cases where Hammer can consider outputs invalid that are actually not, such as special targetnames like `!player`, referencing instance entities directly, classname searches, and probably a few other things. Obviously most of those could be fixed as well, but I still don't really see this point of this animation, it's just annoying. Invalid outputs would still be indicated by  an X at the end of the line, with a solid red color.
